src/assets/locales/de/main.json

```diff
@@ -1997,6 +1997,7 @@
       "decline": "Decline",
       "confirm": "Bestätigen",
       "more": "Mehr",
+      "collapse": "Zusammenbruch",
```

Comment:
haha - that's an ambiguous translation. In this context I guess the correct german term is "Einklappen"